### PR TITLE
chore: Add docker related files for deployment

### PR DESCRIPTION
**Pull request by hugotiburtino**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,9 +39,9 @@
     "eslint": "^8.57.0",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.6",
+    "nodemon": "^3.1.3",
```
Comment (kulla) [inline package.json:42]: I would use `tsx` which has a watch mode via `--watch`

**Review by kulla — CHANGES_REQUESTED** [review]
Maybe you want to merge the other open PRs => Especially https://github.com/serlo/serlo-editor-as-lti-tool/pull/9 should help in development & testing

File: package.json (hunk 1 above)
Comment (hugotiburtino) [reply package.json:46]: Sure, it would be a dependency less.
I'll leave the decision to @LarsTheGlidingSquirrel

Comment (LarsTheGlidingSquirrel) [reply package.json:47]: I will come back to this later

**Pull request merged**